php/composer.lock

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
Do we need to submit this file to the repository? I guess it is generated by IDE according to file `composer.json` to guard against concurrent writes

Reply:
Okay, I'll remove this file from the repository

Reply:
I create a pull request to add all related files into .gitignore, see #280